Add extended error reporting page to documentation (#2082)
JetBrains/intellij-sdk-docs: topics/basics/ide_infrastructure.md

For the latter, reporting is disabled by default — instead, there's an option to disable the plugin causing the exception.

> To let plugins report errors, you can use the following supported solutions:

Reviewer: ```suggestion
To let plugins report errors, the following solutions are available:
```
To avoid "you".
